share/cmake/modules/FindExtPackages.cmake

```diff
@@ -114,6 +114,10 @@ ocio_handle_dependency(  minizip-ng REQUIRED ALLOW_INSTALL
                          MIN_VERSION 3.0.6
                          RECOMMENDED_VERSION 3.0.7
                          RECOMMENDED_VERSION_REASON "Latest version tested with OCIO")
+#minizip-ng requires zlib headers, so we force the dep.
+if(TARGET ZLIB::ZLIB AND TARGET MINIZIP::minizip-ng)
+add_dependencies(MINIZIP::minizip-ng ZLIB::ZLIB)
+endif()
```

Comment:
@SlawekNowy , Stephen Yee's suggestion was to revert the changes here in FindExtPackages.cmake since it should be unnecessary after the change to Installminizip-ng.cmake.